packages/backend/src/messages.ts

```diff
@@ -25,15 +24,17 @@ export default {
   artifact_generated_project_saved_for_future: `${artifact_generated_project} and will be saved for future use.`,
   artifact_generated_module: `The module has been generated.`,
   artifact_generated_files: `The files have been generated.`,
-  artifact_with_name_generated: (artifactName: string) =>
-    `The '${artifactName}' project has been generated.`,
+  artifact_with_name_generated: (artifactName: string) => `The '${artifactName}' project has been generated.`,
   show_progress_button: "Open Output View",
   show_progress_message: "Generating...",
   add_to_workspace: "Open the project in a multi-root workspace",
   open_in_a_new_workspace: "Open the project in a new workspace",
   create_and_close: "Create the project and close it for future use",
-  all_generators_have_been_installed:
-    "All generators have been installed, enjoy your work!",
-  generators_are_being_installed:
-    "Generators are being installed in the background...",
+  all_generators_have_been_installed: "All generators have been installed, enjoy your work!",
+  generators_are_being_installed: "Generators are being installed in the background...",
+  set_default_location: (defaultLocation: string) => `Set generators install location to ${defaultLocation}`,
+  change_owner_for_global: (globalNpmPath: string) => `Change owner ${globalNpmPath} to current user (recommended).\n
+  Admin access required`,
```

Comment:
```suggestion
  change_owner_for_global: (globalNpmPath: string) => `Change owner ${globalNpmPath} to current user (recommended). Admin access required`,
```

Need to remove the \n so the button text appears on modal pop-up on macOS.

Reply:
done

Reply:
I have made 2 more additional changes. It should work now. 
Please test it when you have time.
Thank you for your help.